test_plans/queue_constructors.asciidoc

```diff
@@ -0,0 +1,82 @@
+:sectnums:
+:xrefstyle: short
+
+= Test plan for sycl::queue constructors
+
+This is a test plan for the for new constructors of class `sycl::queue` described in SYCL 2020.
+
+== Testing scope
+
+Tests intend to cover not covered constructors from 4.6.5.1. Queue interface that are not covered in https://github.com/KhronosGroup/SYCL-CTS/blob/SYCL-2020/tests/queue/queue_constructors.cpp
+
+=== Device coverage
+
+Device coverage does not change. All of the tests described below are performed only on the default device that is selected on the CTS command line.
+
+== Tests
+
+=== Check exceptions thrown
+
+Create `syclContext` that doesn't encapsulate selected device. If no other root device is available for context creation, test case is skipped.
+For following constructors use created `syclContext`, selected device or `cts_selector`, `cts_async_handler` and check that synchronous exception with the `errc::invalid` error code is thrown:
+
+* `queue(const context& syclContext, const DeviceSelector& deviceSelector, const property_list& propList = {})`
+* `queue(const context& syclContext, const DeviceSelector& deviceSelector,
+               const async_handler& asyncHandler, const property_list& propList = {})`
+* `queue(const context& syclContext, const device& syclDevice, const property_list& propList = {})`
+* `queue(const context& syclContext, const device& syclDevice,
+               const async_handler& asyncHandler,
+               const property_list& propList = {})`
+
+Note: test with and without both properties `enable_profiling` and `in_order`.
```

Comment:
There is no need to duplicate this test with these properties.  They will have no effect on the error condition that you are testing.

Reply:
Removed cases with properties.